### DISCUSSION TITLE
[Help]: Cannot Connect the Share

### DISCUSSION THREAD

**Discussion by bayramog** [mbentley/docker-timemachine#180] [Help] [answered]
### Describe Your Issue or Question

I deployed docker below compose variables. Network share is shown on my Macbook but when I try to enter TimeMachine share it gave me error. Same for Time Machine backup installation.

### Expected Behavior

I am expecting to enter share folder with my defined username and password.

### Steps to Reproduce

Deploy container with below docker compose.
Wait to see samba share become available on MacBook.
Connect with credentials.
See the TimeMachine share.
Double click to enter in it.

### How You're Launching the Container

```plain
docker run -d --restart=always \
  --name timemachine \
  --net=host \
  -e TM_USERNAME="bayramog" \
  -e TM_GROUPNAME="timemachine" \
  -e PASSWORD="timemachine" \
  -e TM_UID="1000" \
  -e TM_GID="1000" \
  -e SET_PERMISSIONS="false" \
  -e VOLUME_SIZE_LIMIT="0" \
  -v /dev/sda1:/opt/bayramog \
  --tmpfs /run/samba \
  mbentley/timemachine:latest-smb-arm64
```

### Container Logs

```plain
make_connection_snum: '/opt/bayramog' is not a directory, when connecting to [TimeMachine]
make_connection_snum: '/opt/bayramog' is not a directory, when connecting to [TimeMachine]
parse_dfs_path_strict: can't parse hostname from path \homeassistant
```

**Comment by mbentley** [comment, marked as answer]
You're passing a device which is why it says that it isn't a directory:

```
-v /dev/sda1:/opt/bayramog \
```

You should be passing a mounted filesystem instead.
